web-wallet/src/lib/stores/__tests__/walletStore.spec.js

```diff
@@ -322,7 +317,7 @@ describe("walletStore", async () => {
 		});
 
 		it("should expose a method to allow to stake an amount of Dusk", async () => {
-			await walletStore.stake(10);
+			await walletStore.stake(10, gas.price, gas.limit);
 
```

Comment:
As the `dusk-wallet-js` still doesn't support price and limit on a per transaction basis, this is where you should check that the Wallet instance properties are set to `gas.price` and `gas.limit`.